agentless-sms-notifications/nodejs/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "agentless-sms",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "dotenv": "^8.2.0",
```

Comment:
dotenv and getenv don't appear to be used anywhere. Can they be removed from the package file?